kyuubi-main/src/test/scala/org/apache/kyuubi/operation/WithKyuubiServer.scala

```diff
@@ -19,22 +19,31 @@ package org.apache.kyuubi.operation
 
 import java.nio.file.Files
 
-import org.apache.kyuubi.{KyuubiFunSuite, Utils}
+import org.apache.kyuubi.Utils
 import org.apache.kyuubi.config.KyuubiConf
 import org.apache.kyuubi.config.KyuubiConf._
 import org.apache.kyuubi.ha.HighAvailabilityConf.{HA_ZK_ACL_ENABLED, HA_ZK_QUORUM}
 import org.apache.kyuubi.server.KyuubiServer
 import org.apache.kyuubi.zookeeper.{EmbeddedZookeeper, ZookeeperConf}
 
-trait WithKyuubiServer extends KyuubiFunSuite {
+trait WithKyuubiServer extends WithMiniYarnCluster {
 
   protected val conf: KyuubiConf
 
   private var zkServer: EmbeddedZookeeper = _
   private var server: KyuubiServer = _
   private val metastore = Utils.createTempDir()
 
+  override protected val startMiniYarnCluster: Boolean = false
+
   override def beforeAll(): Unit = {
+    super.beforeAll()
+    if (startMiniYarnCluster) {
+      yarnClusterConfig().foreach { case (k, v) =>
+        conf.set(s"spark.hadoop.${k}", v)
```

Comment:
Is it the correct way to integrate with mini yarn cluster?

If not, have we transfer env property like `HADOOP_CONF_DIR` into kyuubi server?

Reply:
cc @yaooqinn

Reply:
It does not work, all application id are local spark application id.

Reply:
missed to set master to yarn, let me retry